### PR TITLE
Channel airspeed measurment over control state in q estimator

### LINKED ISSUE #3264: Control state broke vtol airspeed
The VTOL stack has been using INAV as a default until now. The problem is that the attitude estimator_q does not write airspeed data into the control state topic which at the moment makes it unusable for fixed wing operation.
The question (until we get the new ekf in and running) is if we want to switch to using the ekf for VTOL or if we want the attitude estimator q to subscribe to airspeed and put the value into the control state topic?
@LorenzMeier @yyassa

### PR DESCRIPTION
**Pull request by yyassa**
This fixes #3264

### CODE REVIEW

File: src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp [hunk 1]
```diff
@@ -118,10 +119,10 @@ class AttitudeEstimatorQ
 	bool		_task_should_exit = false;		/**< if true, task should exit */
 	int		_control_task = -1;			/**< task handle for task */
 
-	int		_sensors_sub = -1;
-	int		_params_sub = -1;
+	int		_sensors_sub = -1;	int		_params_sub = -1;
```
Comment (LorenzMeier) [inline src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: What's going on here?

File: src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp [hunk 2]
```diff
@@ -161,6 +162,8 @@ class AttitudeEstimatorQ
 	att_pos_mocap_s _mocap = {};
 	Vector<3>	_mocap_hdg;
 
+	airspeed_s _airspeed = {};
```
Comment (LorenzMeier) [inline src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: :+1: Nice and clean init.

File: src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp [hunk 3]
```diff
@@ -586,6 +599,9 @@ void AttitudeEstimatorQ::task_main()
 
 		ctrl_state.yaw_rate = _lp_yaw_rate.apply(_rates(2));
 
+		/* Airspeed - take airspeed measurement directly here as no wind is estimated */
+		ctrl_state.airspeed = _airspeed.true_airspeed_m_s;
```
Comment (LorenzMeier) [inline src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp]: We should be switching to indicated airspeed, as per https://github.com/PX4/Firmware/issues/3261

**Comment by tumbili** [comment]
@yyassa Format check failed. Please run the fix_code_style script on the att q estimator.

**Comment by tumbili** [comment]
Closing in favour of #3267

**Pull request closed without merging**